packages/bp/src/core/app/bootstrap.ts

```diff
@@ -143,6 +143,7 @@ async function start() {
 
   const resolver = new ModuleResolver(logger)
 
+  // eslint-disable-next-line prefer-const
   let { loadedModules, erroredModules } = await resolveModules(enabledModules, resolver)
```

Comment:
With just the git diff I don't understand why this wasn't switched to a `const`, could you clarify?

Reply:
because erroredModules is reassigned somewhere else, while loadedModules is not

Reply:
```typescript
erroredModules = erroredModules.filter(m => !removedChannels.some(removed => m.entry.location.includes(removed)))
```
Just catched that, sorry.

**PP**: Mutation is not cool.